S2VX.Game/Editor/EditorScreen.cs

```diff
@@ -98,100 +99,116 @@ public class EditorScreen : Screen {
-                        new MenuItem("View")
+                            new MenuItem("Undo (Ctrl+Z)", EditUndo),
+                            new MenuItem("Redo (Ctrl+Shift+Z)", EditRedo)
+                        }
+                    },
+                    new MenuItem("View")
+                    {
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Command Panel (Ctrl+1)", ViewCommandPanel),
-                            }
-                        },
-                        new MenuItem("Playback")
+                            new MenuItem("Editor Interface (H)", ViewEditorUI),
+                            new MenuItem("Command Panel (Ctrl+1)", ViewCommandPanel),
+                        }
+                    },
+                    new MenuItem("Playback")
+                    {
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Play/Pause (Space)", PlaybackPlay),
-                                new MenuItem("Restart (X)", PlaybackRestart),
-                                new MenuItem("Toggle Time Display (T)", PlaybackDisplay),
-                                new MenuItem("Seek Left Tick (Left / MouseWheelUp)", PlaybackSeekLeftTick),
-                                new MenuItem("Seek Right Tick (Right / MouseWheelDown)", PlaybackSeekRightTick),
-                                new MenuItem("Zoom Out Notes Timeline (Ctrl+[)", PlaybackZoomOut),
-                                new MenuItem("Zoom In Notes Timeline (Ctrl+])", PlaybackZoomIn),
-                                new MenuItem("Decrease Beat Snap Divisor (Ctrl+Shift+[)", PlaybackDecreaseBeatDivisor),
-                                new MenuItem("Increase Beat Snap Divisor (Ctrl+Shift+])", PlaybackIncreaseBeatDivisor),
-                                new MenuItem("Decrease Playback Speed (Down, MouseWheelDown over Speed)", PlaybackDecreaseRate),
-                                new MenuItem("Increase Playback Speed (Up,  MouseWheelUp over Speed)", PlaybackIncreaseRate),
-                                new MenuItem("Decrease Volume (MouseWheelDown over Volume)", VolumeDecrease),
-                                new MenuItem("Increase Volume (MouseWheelUp over Volume)", VolumeIncrease),
-                                new MenuItem("Decrease Snapping Divisor (MouseWheelDown over Snap Divisor)", SnapDivisorDecrease),
-                                new MenuItem("Increase Snapping Divisor (MouseWheelUp over Snap Divisor)", SnapDivisorIncrease),
-                            }
-                        },
-                        new MenuItem("Tool")
+                            new MenuItem("Play/Pause (Space)", PlaybackPlay),
+                            new MenuItem("Restart (X)", PlaybackRestart),
```

Comment:
Maybe we should make this not a single letter hotkey since it doesn't seem like it'd be used all that often. Maybe Ctrl+Left?

Reply:
osu! editor uses X to go to beginning, and while I do use it a lot, it's probably more often on accident. Ctrl+Left makes sense.